### ISSUE TITLE
Support presentation remote that sends weird keys / Custom key bindings

### ISSUE THREAD

**Issue by VorpalBlade** [Cimbali/pympress#254]
**Is your feature request related to a problem? Please describe.**
I have a combined presentation remote / touch screen pen that was included with my computer. It sends (according to evtest on Linux) Alt-F20 and Alt-F19 for "forward" and "back" respectively. I cannot find a way to support this in pympress other than rebinding at the OS level using something like AutoKey.

**Describe the solution you'd like**
It would be nice to have support for custom key bindings in pympress, with multiple bindings for the same action (i.e. allow both right arrow and Alt-F20 to go forward). Personally I would be okay with editing a config file for this, I don't need a GUI to configure it.

**Describe alternatives you've considered**
Currently I'm using a pair of [AutoKey](https://github.com/autokey/autokey) scripts to send a right arrow and left arrow when Alt-F20/Alt-F19 are detected. This has multiple downsides:

* There is a noticable delay of maybe 1-2 tenths of a second compared to the output of evtest.
* It is Linux and X11 specific. Any user of Wayland or on another OS could not use this but would need to find specific solutions to their platforms.
* Solutions like AutoKey on Linux with X11 or AutoHotKey on Windows (and got knows what on Mac OS) require a needlessly high level of technical knowledge.

**Additional context**
The presentation remote is a Lenovo Pen Pro, and it was included with a Lenovo ThinkPad P15 Gen1.

**Comment by Cimbali** [comment]
> I cannot find a way to support this in pympress […]

You could likely edit the config file directly. Its actual location should be displayed in the Help > About dialog, and should be `%APPDATA%/pympress.ini` on Windows and `~/.config/pympress` on Linux.

For previous / next pages, the shortcuts you want to extend are `next-page` and `prev-page`, which should be the 2 first ones:

https://github.com/Cimbali/pympress/blob/3696763f534f85061a4437bd912a3c2b71a91767/pympress/share/defaults.conf#L177-L179

Gtk seems to know about [F19](https://docs.gtk.org/gdk3/const.KEY_F19.html) and [F20](https://docs.gtk.org/gdk3/const.KEY_F20.html), so the syntax should be `<alt>F19` and `<alt>F20` respectively:

```ini
[shortcuts]
next-page = Right Down Page_Down space <alt>F20
prev-page = Left Up Page_Up <alt>F19
```

**Comment by VorpalBlade** [comment]
Nice! I was looking in the readme on github and found nothing about editable key bindings. Maybe this bug is really about more obvious documentation in that case? I see now that there is a link to configuration file documentation. But it would have helped lazy or stressed people to have a mention of key bindings directly in the readme with a pointer to the config file.

**Comment by Cimbali** [comment]
Yeah documentation is improvable definitely. Especially making features more discoverable is not very easy
